### DISCUSSION TITLE
Change ressolution of img

### DISCUSSION THREAD

**Discussion by Samuel-Bachorik** [ultralytics/yolov5#4655] [General]
https://github.com/ultralytics/yolov5/blob/fad57c29cd27c0fcbc0038b7b7312b9b6ef922a8/detect.py#L125

Hi at this line in for loop img have shape (1, 3, 480, 640) i need to change it to (1, 3, 384, 512)

Can someone give me TIP where to start ?

**Comment by glenn-jocher** [comment]
@Samuel-Bachorik 
```
python detect.py --img 512
```
